web/src/components/PricesCell/PricesCell.mock.js

```diff
@@ -0,0 +1,4 @@
+// Define your own mock data here:
+export const standard = () => ({
+  prices: [{ id: 42 }, { id: 43 }, { id: 44 }],
+})
```

Comment:
This should get things passing:
```suggestion
export const standard = () => ({
  prices: [
    { id: 42, product: { images: [] } },
    { id: 43, product: { images: [] } },
    { id: 44, product: { images: [] } },
  ],
})```